_includes/examples/dropdown-menu-js.html

```diff
@@ -0,0 +1,57 @@
+/**
+ * Starts the dropdown menu view example component.
+ */
+define([
+    'backbone',
+    'edx-ui-toolkit/js/dropdown-menu/dropdown-menu-view.js'
+    ],
+    function( Backbone, DropdownMenuView ) {
+        'use strict';
+
+        var dropdownModel1 = new Backbone.Model(),
+            dropdownModel2 = new Backbone.Model(),
+            config = {
+                main: {
+                    text: 'dashboard',
+                    url: 'dashboard'
+                },
+                button_label: 'User options dropdown',
+                items: [
+                    {
+                        text: 'Dashboard',
+                        url: 'dashboard'
+                    }, {
+                        text: 'Account',
+                        url: 'account_settings'
+                    }, {
+                        text: 'Profile',
+                        url: 'learner_profile'
+                    }, {
+                        text: 'Sign Out',
+                        url: 'logout'
+                    }
+                ]
+            };
+
+        // The Basic Example
+        dropdownModel1.set(config);
```

Comment:
Nit: I think it would be more interesting if the two examples had different contents. Maybe make the one with the image be the user menu, and then make this one something more generic.